documentation/src/docs/controller-service.md

```diff
@@ -634,7 +639,7 @@ and `updateStream()` operation is performed.
 - Once the update Stream processing starts, it first sets the Stream state to *Updating*.
 - Then, the Stream configuration is updated in the metadata store followed by notifying
 Segment Stores for all _active_ Stream Segments of the Stream, about the change in
-policy. Now the state is reset to *Active*.
+policy. Any Tag related updates causes the Tag reverse indexes to be updated with the newer Tags. Now the state is reset to *Active*.
```

Comment:
we have mentioned indexes earlier and reverse index here. we should perhaps describe how we index it and call it reverse index consistently.

Reply:
improved it.